hpx/parallel/executors/execution_parameters.hpp

```diff
@@ -617,13 +581,7 @@ namespace hpx { namespace parallel { namespace execution
             typedef boost::reference_wrapper<T> wrapper_type;
 
             unwrapper(wrapper_type wrapped_param)
-              : base_member_helper<wrapper_type>(std::move(wrapped_param))
-              , maximal_number_of_chunks_call_helper<T, wrapper_type>(this->member_)
-              , get_chunk_size_call_helper<T, wrapper_type>(this->member_)
-              , mark_begin_execution_call_helper<T, wrapper_type>(this->member_)
-              , mark_end_execution_call_helper<T, wrapper_type>(this->member_)
-              , processing_units_count_call_helper<T, wrapper_type>(this->member_)
-              , reset_thread_distribution_call_helper<T, wrapper_type>(this->member_)
+              : base_member_helper<wrapper_type>(wrapped_param)
```

Comment:
This does not look right, `wrapper_param` is now always copied in the constructor.

Reply:
I understand that the base classes do not need the reference to the derived class anymore (good thinking on that on!). I was referring to the constructor `base_member_helper<wrapper_type>(wrapped_param)` where you have removed the `std::move`s causing the argument to be copied eventually.

Reply:
Right, I missed that.